### ISSUE TITLE
White space beside left margin when window is maximised

### ISSUE THREAD

**Issue by rachelsquirrel** [Automattic/simplenote-electron#349]
When the window is maximised in the Windows app it pushes the left margin for a note to the middle of the screen.

Steps to reproduce
1. Open a note in the Windows app.
2. Maximise the window.
#### What I expected

I expected the contents of the note to be aligned on the left.
#### What happened instead

The left margin was in the middle of the screen.
#### OS version

Windows 10
#### Screenshot / Video

![capture](https://cloud.githubusercontent.com/assets/7946521/17206268/cefe2e4e-54a6-11e6-8510-6ceb5e746fd1.png)

Reported in #2750989-t

**Comment by roundhill** [comment]
This is actually a design implementation, in order to make text of notes more readable. It looks weird with that text though because there's no long lines that wrap in that note.

**Comment by Gen4040** [comment]
Hello!
If this is design implementation, please add some options for user that not need this behavior to have possibility to work on full page width.
Thank you in advance!

**Comment by hedgefield** [comment]
Seconded, the margins on the windows app are crazy wide. This is not the case in either the webapp or the mac app, which look great.

**Comment by salehe** [comment]
One of my notes is a reading list with lots of URLs. It is really hard to follow with some long URLs here and there.

**Comment by supernovia** [comment]
I agree with having shorter lines for readability (except in the case of long urls).

What if just the left margin was removed? It would leave lots of white space on the right, but I think it wouldn't look as bad.

**Comment by SiobhyB** [comment]
This also came up in 818198-z.

**Comment by rachelsquirrel** [comment]
I've got a screenshot of the issue. Reported in 983926-zen

![annotation on 2018-2-3](https://user-images.githubusercontent.com/7946521/37094699-9af8916e-220b-11e8-8f29-21ecaed7c851.png)

**Comment by roundhill** [comment]
> Just a bit wider at large widths (1920px+)

I'm thinking I'll try this out with a PR. It's the easiest of your options :)

**Comment by natanielcz** [comment]
Maybe we should let a user set the width on his own? For example, we could add the scrollbar to change working area width.

**Comment by roundhill** [comment]
Here's an attempt with a break at 1366px, it changes the max width to 1280px:

https://cloudup.com/c8hgWsGM1hE

**Comment by natanielcz** [comment]
Personally, I don't like it. It's really inconvenient for me to read so long lines.

**Comment by jessestu** [comment]
A user mentions this as an issue in 1292890-zen. I'm letting them know it's by design.

I think the margin is good for legibility (although a preference option would be great, too). Is there a way to tweak this design so that the margins are better communicated as a purposeful design?

**Comment by sashastone** [comment]
User's reply to Jesse's message  1292890-zen:

> totally can understand folks not wanting to have lines of unlimited 
> length, but I would argue that's a totally different issue than having 
> the left side margin vary like that.  In other words, keep the left side 
> margin fixed (and close to the list of note names in Windows) but go 
> ahead and wrap things where you all deem best on the right hand side.

**Comment by drw158** [comment]
> I think the margin is good for legibility (although a preference option would be great, too). Is there a way to tweak this design so that the margins are better communicated as a purposeful design?

I think we should add a light background color in the margins so that it looks intentional. I think Ulysses does this via a "page mode", but the default I think is just white space. Do ppl think it's an issue on Ulysses?

We can also use better width breakpoints.

As far as the option goes, I'm in favor of this:

> Include at option to set the max width maybe with 2 options? (full and automatic)

![screen shot 2018-08-02 at 10 59 42 am](https://user-images.githubusercontent.com/618551/43595802-2cc347e6-9643-11e8-893b-98df50594840.png)

![screen shot 2018-08-02 at 10 59 54 am](https://user-images.githubusercontent.com/618551/43595813-34a68874-9643-11e8-8b40-556da7e68219.png)

**Comment by mirka** [comment]
A Line Length setting has been introduced in #815, and will be released soon 🙂
